lib/Elasticsearch/QueryConditionGenerator.php

```diff
@@ -579,45 +579,72 @@ private function injectParameters($template)
 
     private function injectConditions(array $query, array $conditions): array
     {
-        if ([] !== $conditions) {
-            $hasChild = null;
-
-            if (isset($query[self::QUERY_HAS_CHILD])) {
-                // wrap has_child.query into a bool.must to prepare for accepting conditions
-                $query[self::QUERY_HAS_CHILD][self::QUERY] = [
-                    self::QUERY_BOOL => [
-                        self::CONDITION_AND => [
-                            $query[self::QUERY_HAS_CHILD][self::QUERY],
-                        ],
+        if ([] === $conditions) {
+            return $query ?? [];
+        }
+
+        $childType = null;
+        $nestedPath = null;
+
+        if (isset($query[self::QUERY_HAS_CHILD])) {
+            // wrap has_child.query into a bool.must to prepare for accepting conditions
+            $query[self::QUERY_HAS_CHILD][self::QUERY] = [
+                self::QUERY_BOOL => [
+                    self::CONDITION_AND => [
+                        $query[self::QUERY_HAS_CHILD][self::QUERY],
                     ],
-                ];
-                $hasChild = $query[self::QUERY_HAS_CHILD][self::QUERY_TYPE];
-                $nestedBool = &$query[self::QUERY_HAS_CHILD][self::QUERY][self::QUERY_BOOL];
-            }
+                ],
+            ];
+
+            $childType = $query[self::QUERY_HAS_CHILD][self::QUERY_TYPE];
+            $nestedBool = &$query[self::QUERY_HAS_CHILD][self::QUERY][self::QUERY_BOOL];
+        }
 
-            $query = [
+        elseif (isset($query[self::QUERY_NESTED])) {
```

Comment:
CS issue, please run `make cs` to fix.

Reply:
Done. However php-cs-fixer bring even more noise into the PR...

Reply:
Did you run `make cs` or id you use the php-cs-fixer directly?

Reply:
php-cs-fixer directly, as I do not use docker on this particular machine I'm on.

Reply:
I should probably downgrade to the version you use in docker image.

Reply:
Yes, ideally we'd only want related changes here, not changing CS in general.

Reply:
Done.